compiler/ilgen/OMRIlBuilder.cpp

```diff
@@ -992,6 +992,20 @@ OMR::IlBuilder::UnsignedConvertTo(TR::IlType *t, TR::IlValue *v)
    return convertedValue;
    }
 
+TR::IlValue *
+OMR::IlBuilder::Negate(TR::IlValue *v)
+   {
+   TR::DataType dataType = v->getDataType();
+
+   TR::ILOpCodes negateOp = ILOpCode::negateOpCode(dataType);
+   TR_ASSERT(negateOp != TR::BadILOp, "Builder [ %p ] can not negate value %d of type %s", this, v->getID(), dataType.toString());
```

Comment:
So, I'm not someone who regularly points out grammatical errors, but since I happened to notice this one:

"can not" &rarr; "cannot"

Reply:
fixed